.github/workflows/docs.yml

```
@@ -0,0 +1,47 @@
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+
+      - name: Set up Gradle
+        uses: gradle/gradle-build-action@v2
+
+      - name: Build Documentation
+        run: ./gradlew dokkaHtml
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
```

actionlint (1.7.4)

the runner of "aws-actions/configure-aws-credentials@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)